tests/test_knowledge.py

```
@@ -11,7 +11,7 @@
 from sova.db.session import close_db, init_db
 
 
-@pytest.fixture(autouse=True)
+@pytest.fixture(autouse=True, scope="function")
```

Ruff (0.16.1)

[warning] `scope='function'` is implied in `@pytest.fixture()`

Remove implied `scope` argument

(PT003)